tests/test-cases/Group1-Docker-Commands/1-17-Docker-Network-Connect.robot

```diff
@@ -35,82 +35,80 @@ Connect to non-existent network
     Should Contain  ${output}  Error response from daemon: network fakeNetwork not found
 
 Connect containers to multiple networks overlapping
-    ${status}=  Get State Of Github Issue  2498
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-17-Docker-Network-Connect.robot needs to be updated now that Issue #2498 has been resolved
-    Log  Issue \#2498 is blocking implementation  WARN
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network create cross1-network
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network create cross1-network2
-#    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network create cross1-network
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network create cross1-network2
+    Should Be Equal As Integers  ${rc}  0
 
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} pull busybox
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} create --name cross1-container busybox /bin/top
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network connect cross1-network ${containerID}
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network connect cross1-network2 ${containerID}
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${containerID}
-#    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} pull busybox
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} create --name cross1-container busybox /bin/top
```

Comment:
if --network doesn't work we might need to run docker network disconnect as well